README.md

```diff
@@ -468,6 +469,252 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+You can customize the look and behavior of your DirectAnswer, you can create a custom Direct Answer card.
+
+A custom Direct Answer card is given the directAnswer data from the query, the same data the built-in component
```

Comment:
We should have consistency in how we refer to things. The terms "Direct Answer", "DirectAnswer", and "directAnswer" are used here. I think we should just stick to calling it "Direct Answer". You and I understand all 3 phrases have a subtly different meaning. Most people reading this won't have that insight (and it's not too important that they do).